### DISCUSSION TITLE
Versioning scheme?

### DISCUSSION THREAD

**Discussion by felixfontein** [ansible/ansible-lint#1626] [General] [answered]
Right now, ansible-lint seems to use no special versioning theme. Patch versions (x.y.Z with Z > 0) have new features (like new rules), breaking changes (rule behavior change drastically), etc. This is really painful for users.

How about adopting some versioning scheme like semantic versioning, or at least making sure that patch releases are for bugfixes only?

**Comment by ssbarnea** [comment, marked as answer]
I am currently working on a set of changes that should help us respect semantic versioning. Funny bit is that I was doing this before even reading your proposal. Here are the measures planned:

* release only after peer review (last chance for prevent releasing a wrong version/tag number) - already made few minutes ago
* update release drafter config to suggest version numbers based on what the release includes (major/minor/patch) - pending
